vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -60,6 +60,20 @@ class ViewShadowNode : public ShadowNodeBase {
         });
   }
 
+  void dispatchCommand(const std::string &commandId, const folly::dynamic &commandArgs) override {
+    if (commandId == "focus") {
+      if (auto uiManager = GetViewManager()->GetReactContext().NativeUIManager()) {
+        uiManager->focus(m_tag);
+      }
+    } else if (commandId == "blur") {
+      if (auto uiManager = GetViewManager()->GetReactContext().NativeUIManager()) {
+        uiManager->blur(m_tag);
+      }
+    } else {
+      Super::dispatchCommand(commandId, commandArgs);
```

Comment:
For the native code on mobile generated by the codegen we don't seem to call super. Should that be different on windows?

https://github.com/facebook/react-native/blob/master/packages/react-native-codegen/src/generators/components/__tests__/__snapshots__/GeneratePropsJavaDelegate-test.js.snap#L218

Reply:
Hmm, it looks like we call it inconsistently in some view managers. I think it serve a debugging purpose, as we have an debug-time assert() in the parent class.